### PR TITLE
Create `v2` version

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Create `main` branch from existing `v1` branch.
Make needed changes for a new `v2` tag.
Use the opportunity to refactor and remove unneeded input vars.

Tag `v2.0.0` and `v2` in main once this is merged.

Relates to https://github.com/kubewarden/github-actions/issues/48.

## Test

https://github.com/viccuad/verify-image-signatures/actions/runs/4414611656
https://github.com/viccuad/verify-image-signatures/actions/runs/4414611653

### CODE REVIEW

**Comment by viccuad** [comment]
the fail on the release drafter job is expected, as the release-drafter.yaml config is not yet present on `main`.

**Comment by viccuad** [comment]
Dropped https://github.com/kubewarden/github-actions/pull/54/commits/72d2dd15c075c769f808057d297d7bec1d7d58bc as the test has been performed already.

File: .github/release-drafter.yml
Comment (jvanz) [inline .github/release-drafter.yml:1]: \o/

File: README.md
Comment (jvanz) [inline README.md:1]: Thanks for the docs!

**Review by jvanz — APPROVED** [review]

**Pull request merged**